### PR TITLE
eris ports never end. did you think they did?? fool.

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

What a grand and intoxicating innocence...!

## Why It's Good For The Game

https://github.com/discordia-space/CEV-Eris/pull/6137
https://github.com/discordia-space/CEV-Eris/pull/6141
https://github.com/discordia-space/CEV-Eris/pull/6152
https://github.com/discordia-space/CEV-Eris/pull/6156

## Changelog
```changelog
add: manual map changes to add all relevant items to our eris.dmm
del: Excavation suit
add: New moebius hazard suit which spawns everywhere where old excavation suit spawned
add: New hazard suit which spawns in maintenance
add: IH "Tactical Breaching Hammer"
balance: Rebalancing wall smashing mechanic. Make it more depends on stats of the user and item.
add: Added new tactical reload (empty gun -> click on mag -> loaded gun)
```

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
fine, good to merge

File: code/game/objects/items/weapons/tools/hammer.dm [hunk 1]
```diff
@@ -152,3 +152,18 @@
 		playsound(src, 'sound/machines/hiss.ogg', 50, 0, 0)
 		user.throw_at(target, get_dist(target, user), 1, user)
 		T.stop()
+
+/obj/item/weapon/tool/hammer/IH
```
Comment (Jamini) [inline code/game/objects/items/weapons/tools/hammer.dm:156]: Why are we porting this? We don't need this meme item

File: code/modules/research/xenoarchaeology/tools/anomaly_suit.dm [hunk 2]
```diff
@@ -1,14 +1,53 @@
...
-	armor = list(melee = 25, bullet = 10, energy = 50, bomb = 0, bio = 100, rad = 100)
-	allowed = list(/obj/item/device/lighting/toggleable/flashlight,/obj/item/weapon/tank,/obj/item/device/suit_cooling_unit)
-
-/obj/item/clothing/head/space/anomaly
-	name = "Expedition hood"
-	desc = "A pressure resistant expedition hood partially capable of insulating against exotic alien energies."
-	icon_state = "cespace_helmet"
-	item_state = "cespace_helmet"
-	armor = list(melee = 25, bullet = 10, energy = 50, bomb = 0, bio = 100, rad = 100)
+// hazard suit, maint version. Replaces old excavation suit.
```
Comment (Jamini) [inline code/modules/research/xenoarchaeology/tools/anomaly_suit.dm:1]: This is ported in the other Port PR

**Review by Jamini — CHANGES_REQUESTED** [review]
Bear, the hazard suit is already in the other PR that I have up. This is going to conflict.
We also really don't need or want the IH memehammer.

File: code/modules/research/xenoarchaeology/tools/anomaly_suit.dm (hunk 2 above)
Comment (bearrrrrrrr) [reply code/modules/research/xenoarchaeology/tools/anomaly_suit.dm:1]: does the other PR have the appropriate map changes for it, as well??

**Comment by bearrrrrrrr** [comment]
whatever

**Pull request closed without merging**